Entities/QuestionOptions.cs

```diff
@@ -0,0 +1,8 @@
+namespace X.Kwiz.Api.Entities;
+public class QuestionOption
+{
+    public string Content {get;set;}
+    public bool IsCorrect {get;set;}
+    public bool IsRequired {get;set;}
+    public string Explanation {get;set;}
+}
```

Comment:
```suggestion
    public Guid QuestionId { get; set; }
    public QuizQuestion Question { get; set; }
}
```

we should add one to many navaition between question and options